src/server/db/uploads.js

```diff
@@ -40,6 +40,18 @@ async function uploadsForAgency (agency_id, period_id) {
     .orderBy('created_at', 'desc')
 }
 
+async function uploadsForPeriod (period_id) {
```

Comment:
is this method still useful? doesn't look like it's getting called anywhere else in this PR